frontend/src/metabase/admin/settings/slack/selectors.ts

```diff
@@ -1,18 +1,22 @@
-import { State } from "metabase-types/store";
-import { SlackSettings } from "metabase-types/api";
+import { getSetting } from "metabase/selectors/settings";
+import type { State } from "metabase-types/store";
+import type { SlackSettings } from "metabase-types/api";
 
 export const getSlackSettings = (state: State): SlackSettings => {
-  return state.settings.values;
+  return {
+    "slack-app-token": getSetting(state, "slack-app-token"),
+    "slack-files-channel": getSetting(state, "slack-files-channel"),
+  };
 };
```

Comment:
`SlackSettings` return type only contains these two settings, so it doesn't make sense to return the full settings object here